modules/ROOT/pages/sap/sap-s4hana-cloud-connector-reference-2-0.adoc

```diff
@@ -418,7 +440,7 @@ Executes an update operation within the connected SAP S/4HANA instance. This ope
 |===
 
 [[KeyStore]]
-=== Key Store
+=== Keystore
```

Comment:
For the Type field "store" should probably be "keystore."

Reply:
I will leave this one out, since it's something generated from source code which we cannot modify, so this would be "reverted" in the next release.